pytest.ini

```diff
@@ -1,14 +1,29 @@
 [pytest]
-env =
-    TESTING_CLAWBACK_CLI=TRUE
-    CHIA_ROOT=~/.chia/simulator/main
+; logging options
 log_cli = False
 addopts = --verbose --tb=short -p no:monitor
 log_level = WARNING
 console_output_style = count
 log_format = %(asctime)s %(name)s: %(levelname)s %(message)s
 asyncio_mode = strict
+markers=benchmark
 testpaths = tests
 filterwarnings =
+    error
     ignore:ssl_context is deprecated:DeprecationWarning
+    ignore:Implicitly cleaning up:ResourceWarning
+    ignore:unclosed <socket\.socket:ResourceWarning
+    ignore:Unclosed client session:ResourceWarning
+    ignore:unclosed file:ResourceWarning
+    ignore:unclosed transport:ResourceWarning
+    ignore:coroutine '.*' was never awaited:RuntimeWarning
     ignore:`coin_solutions` is now `coin_spends`:UserWarning
+    ignore:Exception ignored in:pytest.PytestUnraisableExceptionWarning
+    ignore:cannot collect test class:pytest.PytestCollectionWarning
+    ignore:The loop argument is deprecated since Python 3\.8, and scheduled for removal in Python 3\.10.:DeprecationWarning
+    ignore:The distutils package is deprecated:DeprecationWarning
+    ignore:There is no current event loop:DeprecationWarning
+    ignore:getargs. The 'u' format is deprecated. Use 'U' instead.:DeprecationWarning
+    ignore:BackendFinder.find_spec\(\) not found; falling back to find_module\(\):ImportWarning
+    ignore:BackendLoader.exec_module\(\) not found; falling back to load_module\(\):ImportWarning
+    ignore:pkg_resources is deprecated as an API:DeprecationWarning
```

Comment:
Are these all required?  Generally, these exceptions are added for things we need to clean up or where we are waiting for upstream fixes.  We don't want this list any bigger than it has reason to be and would prefer, when possible, to just fix the issue.  I understand that we use blockchain here so it may be that many of these issues pass through and need ignored here.  Just making sure they've been considered individually before being ignored.